bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -136,6 +143,13 @@ message GetBookieInfoRequest {
     optional int64 requested = 1;
 }
 
+message SyncRequest {
+    required int64 ledgerId = 1;
+    required bytes masterKey = 2;
+    required int64 firstEntryId = 3;
```

Comment:
why do you need first and last entry ids?

sync means flushing any outstanding requests and the response should tell you where does it sync to, no?